doc/api/console.md

```diff
@@ -277,6 +277,20 @@ Defaults to `2`. To make it recurse indefinitely, pass `null`.
 Defaults to `false`. Colors are customizable; see
 [customizing `util.inspect()` colors][].
 
+### console.dirxml(...data)
+<!-- YAML
+added: v8.0.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/17128
+    description: "`console.dirxml` now calls `console.dir` for each argument."
+-->
+* `...data` {any}
+
+This method calls `console.dir()` with default options for each argument it
+receives. See [`console.dir()`][] for more details about said defaults.
```

Comment:
This will require some changes.

Reply:
Damn, sorry about that. Fixing right now and squashing.

Reply:
Fixed!